File: official/docs/php/v6/batches/label.php
<?php

$client = new \EasyPost\EasyPostClient(getenv('EASYPOST_API_KEY'));

$batch = $client->batch->retrieve('batch_...');

$batchWithLabel = $client->batch->label(
    $batch->id,
    ['file_format' => 'PDF']
);

echo $batchWithLabel;

File: official/docs/csharp/v6/batches/label.cs
using System;
using System.Collections.Generic;
using System.Threading.Tasks;
using EasyPost;
using Newtonsoft.Json;

namespace EasyPostExamples
{
    public class Examples
    {
        public static async Task Main()
        {
            var client = new EasyPost.Client(new EasyPost.ClientConfiguration("EASYPOST_API_KEY"));

            EasyPost.Parameters.Batch.GenerateLabel parameters = new()
            {
                FileFormat = "PDF",
            };

            EasyPost.Models.API.Batch batch = await client.Batch.GenerateLabel("batch_...", parameters);

            Console.WriteLine(JsonConvert.SerializeObject(batch, Formatting.Indented));
        }
    }
}
